tools/homebrew-start/redis.conf

```diff
@@ -160,7 +160,7 @@ loglevel notice
 # Specify the log file name. Also the empty string can be used to force
 # Redis to log on the standard output. Note that if you use standard
 # output for logging but daemonize, logs will be sent to /dev/null
-logfile <LRR_DATA_DIRECTORY>/log/redis.log
+logfile "<LRR_DATA_DIRECTORY>/log/redis.log"
```

Comment:
As for the log situation, `redis` had problems with the path since it contained whitespace (`Application Support`). It works when quoted, so we don’t have to escape anything :)